packages/synthetics-sdk-broken-links/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "synthetics-sdk-broken-links",
+  "version": "1.0.0",
```

Comment:
nock this down to 0.1.0 until we go to GA - this will give us some room to be able to include broken changes before launch.

Reply:
done.